src/pkg/gc/gc_test.go

```diff
@@ -29,9 +30,37 @@ func (d *FakeDiskSpaceFetcher) GetUsedDiskSpaceInPercents() (int, error) {
 	return d.counter, nil
 }
 
-type testResponseMap map[string]httpMethodAndResponseMap
-type httpMethodAndResponseMap map[string]parameterAndResponseMap
-type parameterAndResponseMap map[string]string
+type testResponseMap map[string][]response
+
+type response struct {
+	method   string
+	params   string
+	response string
+}
+
+type containerListInfo struct {
+	Id    string `json:"Id"`
```

Comment:
Yeah, I know but I like to have this to have it more verbose and inline. In one case (`Status`) this isn't true so it would be weird IMHO for that to stand out